tokens/properties/components/LookupMenu.json

```diff
@@ -0,0 +1,88 @@
+{
+	"wikit-LookupMenu": {
+		"background-color": {
+			"value": "{background.color.base.default.value}"
+		},
+		"border" : {
+			"style" : {
+				"value": "{border.style.base.value}"
+			},
+			"width" : {
+				"value": "{border.width.thin.value}"
+			},
+			"radius" : {
+				"value": "{border.style.base.value}"
+			},
+			"color" : {
+				"value": "{border.color.base.default.value}"
+			}
+		},
+		"box-shadow": {
+			"value": "{box-shadow.elevation.menu.value}"
+		},
+		"item" : {
+			"padding-vertical": {
+				"value": "{dimension.spacing.small.value}"
+			},
+			"padding-horizontal": {
+				"value": "{dimension.spacing.medium.value}"
+			},
+			"hover" : {
+				"background-color": {
+					"value" : "{background.color.base.hover.value}"
+				}
+			},
+			"active" : {
+				"color": {
+					"value": "{font.color.progressive.value}"
+				},
+				"background-color":{ 
+					"value" : "{background.color.base.active.value}"
+				}
+			},
+			"transition-duration" : {
+				"value": "{transition.interaction.menu.duration.value}"
+			},
+			"transition-property" : {
+				"value": "{transition.interaction.menu.property.value}"
+			},
+			"transition-timing-function" : {
+				"value": "{transition.timing-function.ease.value}"
+			}
+		},
+		"label" : {
```

Comment:
Shouldn't this be nested inside "item"? It's the label of a menu item, not the label of the menu.

Reply:
yes